Week3/Practice Code/Abdullah/Week3_Project/src/middlewares/validationMiddleware.js

```diff
@@ -30,5 +30,30 @@ export const validatorTask = (req, res, next) => {
         return res.status(400).json({ message: 'Invalid priority. Must be one of: low, medium, high' });
     }
 
+    next(); 
+};
+
+
+ export const  updateValidatuion = (req, res, next) => {
```

Comment:
code for this middleware is not correct. As when any of the fields such as title is not passed it throws an error like "Title must be a string".  It should not force user to enter all fields as it is an update method.

Reply:
It is working correctly at my end .If i am not writing any title it is throwing title from the creation or from the last update . or if i am not adding a string it showing "Title must be a string" error